src/test/java/org/folio/circulation/rules/Text2DroolsTest.java

```diff
@@ -482,6 +482,24 @@ void missingLoanPolicy() {
       matches("Policy missing after ':'", 3, 8));
   }
 
+  @Test
+  void missingColonAfterPolicy() {
```

Comment:
Why is colon missing 'after' a policy? I thought it should go before policies and after criteria.

Reply:
thanks, done